mmv1/products/compute/RegionSecurityPolicyRule.yaml

```diff
@@ -0,0 +1,216 @@
+    path: 'error/errors'
+    message: 'message'
+examples:
+  - !ruby/object:Provider::Terraform::Examples
+    name: 'region_security_policy_rule_basic'
+    primary_resource_id: 'policy_rule'
+    min_version: 'beta'
+    vars:
+      sec_policy_name: 'policyruletest'
+  - !ruby/object:Provider::Terraform::Examples
+    name: 'region_security_policy_rule_multiple_rules'
+    primary_resource_id: 'policy_rule_one'
+    min_version: 'beta'
+    vars:
+      sec_policy_name: 'policywithmultiplerules'
+  - !ruby/object:Provider::Terraform::Examples
+    name: 'region_security_policy_rule_with_network_match'
+    # it needs to run synchronously because a region can have only one google_compute_network_edge_security_service.
+    # there is a robust handwritten test which covers this scenario. 
+    skip_test: true
+    primary_resource_id: 'policy_rule_network_match'
+    min_version: 'beta'
+    vars:
+      sec_policy_name: 'policyfornetworkmatch'
+parameters:
+  - !ruby/object:Api::Type::String
+    name: 'region'
+    description: |
+      The Region in which the created Region Security Policy rule should reside.
+    url_param_only: true
+    required: true
+  - !ruby/object:Api::Type::String
+    name: 'security_policy'
+    required: true
+    immutable: true
+    description: |
+      The name of the security policy this rule belongs to.
+    url_param_only: true
+properties:
+  - !ruby/object:Api::Type::String
+    name: 'description'
+    description: |
+      An optional description of this resource. Provide this property when you create the resource.
+  - !ruby/object:Api::Type::Integer
+    name: 'priority'
+    description: |
+      An integer indicating the priority of a rule in the list.
+      The priority must be a positive value between 0 and 2147483647.
+      Rules are evaluated from highest to lowest priority where 0 is the highest priority and 2147483647 is the lowest priority.
+    required: true
+    immutable: true
+  - !ruby/object:Api::Type::NestedObject
+    name: 'match'
+    description: |
+      A match condition that incoming traffic is evaluated against.
+      If it evaluates to true, the corresponding 'action' is enforced.
+    properties:
+      - !ruby/object:Api::Type::Enum
+        name: 'versionedExpr'
+        description: |
+          Preconfigured versioned expression. If this field is specified, config must also be specified.
+          Available preconfigured expressions along with their requirements are: SRC_IPS_V1 - must specify the corresponding srcIpRange field in config.
+        values:
+          - :SRC_IPS_V1
+        default_value: :SRC_IPS_V1
+      - !ruby/object:Api::Type::NestedObject
+        name: 'config'
+        description: |
+          The configuration options available when specifying versionedExpr.
+          This field must be specified if versionedExpr is specified and cannot be specified if versionedExpr is not specified.
+        required: true
```

Comment:
It looks like the field `config` is only required when `versionedExpr` is specified. `required: true` is not always true.

Reply:
Done